src/order/category/LinearOrder.lean

```diff
@@ -6,10 +6,16 @@ Authors: Johan Commelin
 
 import order.category.PartialOrder
 
-/-! # Category of linearly ordered types -/
+/-!
+# Category of linear orders
```

Comment:
```suggestion
# Category of linearly ordered types
```
in my opinion - if not then at least the docstring on `LinearOrder` should match!